access-modifier-annotation/src/main/java/org/kohsuke/accmod/restrictions/ProtectedExternally.java

```diff
@@ -47,6 +49,9 @@
 public class ProtectedExternally extends None {
 
     @Override
+    @SuppressFBWarnings(
+            value={"RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE","RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", "NP_LOAD_OF_KNOWN_NULL_VALUE"},
+            justification="something in this class confuses spotbugs, I can't see anything redundant and the error line is the catch block")
```

Comment:
Me neither. It is a historical code anyway, can be revisited later

Reply:
There is a known bug in SpotBugs that was resolved sort of recently involving `try`-with-resources and null checks.